src/snowcli/cli/common/sql_execution.py

```diff
@@ -23,13 +25,32 @@ def _conn(self):
     def _log(self):
         return logging.getLogger(__name__)
 
+    def _execute_string(
+        self,
+        sql_text: str,
+        remove_comments: bool = False,
+        return_cursors: bool = True,
+        cursor_class: SnowflakeCursor = SnowflakeCursor,
+        **kwargs,
+    ) -> Iterable[SnowflakeCursor]:
+        """
+        This is a custom implementation of SnowflakeConnection.execute_string that returns generator
+        instead of list. In case of executing multiple queries are executed one by one. This mean we can
+        access result of previous queries while evaluating next one. For example, we can print the results.
+        """
+        self._log.info("Executing %s", sql_text)
```

Comment:
Was it intentional to change log level (debug -> info)?

Reply:
I don't have strong arguments for this change. I'm fine with going back to debug.

Reply:
Then IMO we should go back to `debug` as this change wasn't in plan for this task.

Reply:
Fixed